pages/api/sendgrid.js

```diff
@@ -0,0 +1,50 @@
+import sendgrid from "@sendgrid/mail";
+
+sendgrid.setApiKey(process.env.SENDGRID_API_KEY);
+
+async function sendEmail(req, res) {
+  try {
+    // console.log("REQ.BODY", req.body);
+    await sendgrid.send({
+      to: [`${req.body.email}`, "monika.moj74@googlemail.com"],
+      from: {
+        name: `${req.body.fullname}`,
+        email: "monika.moj74@googlemail.com",
+      },
+      subject: `[Lead from website] :${req.body.subject}`,
+      html: `<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+      <html lang="en">
+      <head>
+        <meta charset="utf-8">
+      
+        <title>The HTML5 Herald</title>
+        <meta name="description" content="The HTML5 Herald">
+        <meta name="author" content="SitePoint">
+      <meta http-equiv="Content-Type" content="text/html charset=UTF-8" />
+      
+        <link rel="stylesheet" href="css/styles.css?v=1.0">
+      
+      </head>
+      
+      <body>
+        <div class="img-container" style="display: flex;justify-content: center;align-items: center;border-radius: 5px;overflow: hidden; font-family: 'helvetica', 'ui-sans';">              
+              </div>
+              <div class="container" style="margin-left: 20px;margin-right: 20px;">
+              <h3>You've got a new mail from ${req.body.fullname}, their email is: ✉️${req.body.email} </h3>
+              <div style="font-size: 16px;">
+              <p>Message:</p>
+              <p>${req.body.message}</p>
+              <br>
+              </div>
+              </body>
+      </html>`,
+    });
+  } catch (error) {
+    // console.log(error);
+    return res.status(error.statusCode || 500).json({ error: error.message });
```

Comment:
Please note, that this **could** be potentially dangerous since you are forwarding the error message to the client.

I do not know what kind of errors sendgrid might throw. But say it's something like:
> "API-Key **12432** did not work"

If that is send to the client, you've basically given away your login credentials. That'd be not good.